client/src/utils/date/index.js

```diff
@@ -0,0 +1,16 @@
+const monthNames = [
+    "January", "February", "March",
+    "April", "May", "June", "July",
+    "August", "September", "October",
+    "November", "December"
+  ];
+
+const simpleDate = (dateStr) => {
+    const date = new Date(dateStr);
+    const day = date.getDate()
+    const monthIndex = date.getMonth();
+    const year = date.getFullYear();
+    return `${day  } ${  monthNames[monthIndex]  } ${  year}`;
```

Comment:
It will break if the date string is not valid. Make sure you are using doing error handling in the parts where you will be using this.

Reply:
Taken care.